fr-FR.lproj/Localizable.strings

```diff
@@ -0,0 +1,369 @@
+"Set kernel task info.\n" = "Définir les informations sur la tâche du kernel.\n"
+"Unable to make my task a platform task." = "Impossible de transformer ma tâche en tâche de plate-forme."
+"Unable to make my codesign blob a platform blob." = "Impossible de transformer mon objet de code en objet de plateforme."
+"Exporting kernel task port..." = "Exportation du port de la tâche du kernel..."
+"Unable to export TFP0." = "Impossible d'exporter TFP0."
+"Exported kernel task port.\n" = "Impossible d'exporter TFP0....\n"
+"Unexporting kernel task port..." = "Port de tâche du kernel de non exportation..."
+"Unable to unexport TFP0." = "Impossible d'exporter TFP0."
+"Unexported kernel task port.\n" = "Port de tâche du kernel non exporté.\n"
+"Writing a test file to UserFS..." = "Écrire un fichier de test sur UserFS..."
+"Dumping APTicket..." = "Suppression APTicket..."
+"Unable to dump APTicket." = "Impossible de supprimer APTicket."
+"Dumped APTicket.\n" = "APTicket supprimés.\n"
+"Unlocking nvram..." = "Débloquer nvram..."
+"Unable to unlock nvram." = "Impossible de déverrouiller nvram."
+"Unable to print nvram variables." = "Impossible d'imprimer les variables nvram."
+"Setting boot-nonce..." = "Réglage du boot-nonce..."
+"Unable to set boot nonce." = "Impossible de régler la méthode de signature au démarrage."
+"Unable to synchronize boot nonce." = "Impossiblede synchroniser la méthode de signature au démarrage."
+"Unable to print new nvram variables." = "Impossible d'afficher de nouvelles variables nvram"
+"Locking nvram..." = "Verrouillage nvram..."
+"Unable to lock nvram." = "Impossible de vérrouiller nvram."
+"Overwrote boot nonce.\n" = "Écraser la méthode de signature au démarrage.\n"
+"Logging slide..." = "Diapositive de journalisation..."
+"Unable to clean old kernel slide log." = "Impossible de nettoyer l'ancien journal des logs du kernel."
+"Unable to log kernel slide." = "Impossible de se connecter à la diapositive du noyau."
+"Logged slide.\n" = "Slide coonecté.\n"
+"Logging ECID..." = "Connexion ECID..."
+"Logged ECID.\n" = "Connecté ECID.\n"
+"Disabling Auto Updates..." = "Désactivation des mises à jour automatiques..."
+"Unable to disable software update badge." = "Impossible de désactiver le badge de mise à jour logicielle."
+"Disabled Auto Updates.\n" = "Mises à jour automatiques désactivées.\n"
+"Enabling Auto Updates..." = "Mises à jour automatiques activées..."
+"Unable to enable software update badge." = "Impossible d'activer le badge de mise à jour logicielle."
+"Enabled Auto Updates.\n" = "Activé Mises à jour automatiques.\n"
+"Remounting RootFS..." = "Remontage RootFS..."
+"Unable to open RootFS." = "Impossible d'ouvrir RootFS."
+"Unable to get vnode for RootFS." = "Impossible d'obtenir vnode pour RootFS."
+"Unable to get mount info for RootFS." = "Impossible d'obtenir les informations de montage pour RootFS."
+"Unable to copy system snapshot." = "Impossible de copier l'instantané du système."
```

Comment:
Impossible de copier le RootFS.

Reply:
"Unable to get mount info for RootFS." = " information montage pour RootFS indisponible."
"Unable to copy system snapshot." = "copie snapshot système Impossible."